src/operationresult.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_OPERATIONRESULT_H
+#define BITCOIN_OPERATIONRESULT_H
+
+#include <util/translation.h>
+
+class OperationResult
```

Comment:
In f198d27ef7f80b6b1c03f388eae216402f1ff424 "introduce OperationResult and CallResult classes"

Since `OperationResult` is only used by `CallResult` now, what do you think about consolidating the two classes?

Reply:
I left it because we can make use of it in other functions that only return a bool and the error string without the extra object. Like for example `CConnman::BindListenPort`, `Chain::broadcastTransaction` and `CWallet::SubmitTxMemoryPoolAndRelay`, `BerkeleyEnvironment::Open`, `BerkeleyDatabase::Verify` etc.

I mean, having the class available would let others use it on different PRs if they are touching any similar function signature.

Saying that, if you are still against merging it without a connected use-case, can just remove it and re-introduce it later when we modify any of those functions as well (or.. make an extra small commit that connects it to one of those functions just to present the use-case for it).